client/src/components/Carousel.js

```diff
@@ -23,7 +23,7 @@ const Carousel = () => {
           return current + 1;
         }
       });
-    }, 5000);
+    }, 10000);
```

Comment:
why you are increasing the time?? if it's for giving more time to images to load this is not going to work. why?? images used in internet should be lite on size

Reply:
I will change it to 5000.